From: Jiang Xin <worldhello.net@gmail.com>
Subject: [PATCH] t/debian-fix-gl-admindir-perms

Set $GL_ADMINDIR perms to 0750, so that group users (such as www-data)
can use gitolite auth service.

Signed-off-by: Jiang Xin <worldhello.net@gmail.com>

---
 src/gl-install |    4 ++--
 1 个文件被修改，插入 2 行(+)，删除 2 行(-)

diff --git a/src/gl-install b/src/gl-install
index ac9e84e..31783f4 100755
--- a/src/gl-install
+++ b/src/gl-install
@@ -26,11 +26,11 @@ setup_environment();
 
 # mkdir $REPO_BASE, $GL_ADMINDIR if they don't already exist
 wrap_mkdir($REPO_BASE);
-wrap_mkdir( $GL_ADMINDIR, 0700 );
+wrap_mkdir( $GL_ADMINDIR, 0750 );
 
 # mkdir $GL_ADMINDIR's subdirs
 for my $dir (qw(conf keydir logs hooks hooks/common hooks/gitolite-admin)) {
-    wrap_mkdir( "$GL_ADMINDIR/$dir", 0700 );
+    wrap_mkdir( "$GL_ADMINDIR/$dir", 0750 );
 }
 
 # hooks must be propagated to all the repos in case they changed
-- 
tg: (f73e3d7..) t/debian-fix-gl-admindir-perms (depends on: upstream)
